interfaces/udev/spec.go

```diff
@@ -110,8 +110,13 @@ func (spec *Specification) TagDevice(snippet string) {
 		// SUBSYSTEM=="subsystem" is for subsystems (the top directories in /sys/class). Not for devices.
 		// When loaded, they send an ADD event
 		// snap-device-helper expects devices only, not modules nor subsystems
-		spec.addEntry(fmt.Sprintf("TAG==\"%s\", SUBSYSTEM!=\"module\", SUBSYSTEM!=\"subsystem\", RUN+=\"%s/snap-device-helper %s\"",
-			tag, dirs.DistroLibExecDir, tag), tag)
+		if useOldCall() {
```

Comment:
we don't really want to call "snap --version" again and again from here

Reply:
We cache it.